openpower/package/sb-signing-framework/sb-signing-framework.mk

```diff
@@ -0,0 +1,27 @@
+################################################################################
+#
+#  signframework
+#
+################################################################################
+
+SB_SIGNING_FRAMEWORK_SITE ?= $(call github,open-power,sb-signing-framework,$(SB_SIGNING_FRAMEWORK_VERSION))
+
+SB_SIGNING_FRAMEWORK_LICENSE = Apache-2.0
+SB_SIGNING_FRAMEWORK_LICENSE_FILES = LICENSE
+SB_SIGNING_FRAMEWORK_VERSION ?= master
```

Comment:
master -> commit # 

You don't always want to pick master because a master change could break all of op build; instead test jobs verify a new update to the project, and issue pull requests to change the *VERSION pointer